service/resource/configmap/update_test.go

```diff
@@ -441,9 +441,11 @@ func Test_Resource_ConfigMap_Reload(t *testing.T) {
 	configMapNamespace := "monitoring"
 
 	var receivedMessage *http.Request = nil
+	var reloadCount int = 0
```

Comment:
This test would be easier to understand if it were split into test cases instead of one long test that covers multiple assertions.

Reply:
I have split up the tests.

Reply:
I can only see 1 commit still? 😕

Reply:
Ammended the commit